cli/src/app.rs

```diff
@@ -188,16 +188,33 @@ pub fn add_subcommands(command: Command) -> Command {
                 .about("Manage authentication, registration, and API keys")
                 .arg_required_else_help(true)
                 .subcommand_required(true)
-                .subcommand(Command::new("register").about("Register a new account"))
                 .subcommand(
-                    Command::new("login").about("Login to an existing account").arg(
-                        Arg::new("reauth")
-                            .action(ArgAction::SetTrue)
-                            .short('r')
-                            .long("reauth")
-                            .help("Force a login prompt"),
+                    Command::new("register").about("Register a new account").arg(
+                        Arg::new("token-name")
+                            .action(ArgAction::Set)
+                            .short('n')
+                            .long("token-name")
```

Comment:
It actually took me a second to figure out what "token-name" means in this context. I confused it with the actual locksmith token for a second.

I probably got myself confused a bit, but it might be worth taking into account and maybe expanding the help a bit?

Reply:
Yeah. "token-name" was the best I could come up with. We definitely use many different names for the token itself (e.g., "token", "API Key", "API Access Token", etc.)...

I'll try to make the help text more descriptive.

Reply:
I think token name is fine, I'd just make the help text more verbose.